library/src/main/java/com/mvc/imagepicker/ImagePicker.java

```diff
@@ -254,7 +274,8 @@ public static InputStream getInputStreamFromResult(Context context, int requestC
             if (isCamera) {     /** CAMERA **/
                 selectedImage = getUriForFile(context, imageFile);
             } else {            /** ALBUM **/
-                selectedImage = imageReturnedIntent.getData();
+                selectedImage = getUriForFile(context,
+                        new File(getRealPathFromURI(context, imageReturnedIntent.getData())));
```

Comment:
This will cause an exception if that URI is for example: content://com.google.android.apps.photos.contentprovider/1/1/mediakey%3A%2Flocal%253A44d10898-1ad8-4a47-922d-8d75f15b2d18/ORIGINAL/NONE/1436044117